x-pack/plugins/graph/server/routes/explore.ts

```diff
@@ -27,6 +27,13 @@ export function registerExploreRoute({
   router.post(
     {
       path: '/internal/graph/graphExplore',
+      security: {
+        authz: {
+          enabled: false,
+          reason:
+            'This route is opted out from authorization because uses the current user authorizations.',
```

Comment:
```suggestion
            'This route delegates authorization to the scoped ES client.',
```

I guess that was the meaning? 
> current user authorizations

User authorization sounds a bit ubiquitous to me

Reply:
fixed in aaf263bb1a35c0ff1c145195d70a739a46082b08